test/pipelines/validate_test.exs

```diff
@@ -0,0 +1,43 @@
+defmodule Pipelines.ValidateTest do
+  use ExUnit.Case, async: false
+
+  @valid %{
+    title: "title",
+    author: "data"
+  }
+  @invalid_missing %{
+    title: "title"
+  }
+  @invalid_nil %{
+    title: "title",
+    author: nil
+  }
+
+  test "ValidateItem item with required fields are not dropped, item returned unchanged" do
+    Application.put_env(:crawly, :item, [:title, :author])
+    pipelines = [Crawly.Pipelines.Validate]
+    item = @valid
+    state = %{}
+
+    {item, _state} = Crawly.Utils.pipe(pipelines, item, state)
+    assert item == @valid
+  end
+
+  test "ValidateItem items with missing required fields are dropped" do
+    Application.put_env(:crawly, :item, [:title, :author])
+    pipelines = [Crawly.Pipelines.Validate]
+    item = @invalid_missing
+    state = %{}
+
+    {false, _state} = Crawly.Utils.pipe(pipelines, item, state)
+  end
+
+  test "ValidateItem required item fields with nil are dropped" do
```

Comment:
What if we will rewrite test names in the following way (pattern):

`Drops items with required field set to nil`?

My idea is that, with this ValidateItem is already stated in the module name. Maybe lets not repeat ourselves. And another point, I like something like a command language. E.g. it explains what behaviour this test assures

Reply:
Yup I agree.

So test names should begin with a verb?

Reply:
test names have been adjusted